### DISCUSSION TITLE
Change format for list of task in Daily Note

### DISCUSSION THREAD

**Discussion by y0j0** [davish/obsidian-full-calendar#422] [Feature Requests]
In [this feature request ](https://github.com/davish/obsidian-full-calendar/issues/63) you added possibility to list tasks from Full Calendar in current daily note. It's great but it will list the tasks in following format

- [ ] Test [startTime:: 11:00]  [endTime:: 12:00]

<img width="382" alt="image" src="https://user-images.githubusercontent.com/2878125/220075810-27419ad4-d520-4038-8bab-a94ed9a8ed0d.png">

To be able to write and read such list more easily it would be great to use just format like

- [ ] 09:30 - 10:00 Setup for work
- [ ] 10:00 - 11:00 Create new notes for #article review

It could be potentially used then also in [Day Planner](https://github.com/lynchjames/obsidian-day-planner) plugin that works with this format.

After adding a new item to calendar the output list is also not correctly ordered. The new task is displayed as the first although it is added as the last in the day.

Thanks

**Comment by stefanheyder** [comment]
Hopping onto this issue to say that a custom format would also allow us to better combine full-calendar with [Obsidian tasks](https://github.com/obsidian-tasks-group/obsidian-tasks).
